ibl_to_nwb/datainterfaces/iblsortingextractor.py

```diff
@@ -80,9 +83,15 @@ def __init__(self, session: str, cache_folder: Optional[DirectoryPath] = None):
                 spike_count="spike_count",
                 firing_rate="firing_rate",
                 label="label",
+                uuid="uuid",
+                cluster_id="cluster_id",
             )
+            
+            cluster_metrics = clusters['metrics'].reset_index(drop=True).join(pd.DataFrame(clusters['uuids']))
+            cluster_metrics.rename(columns={'uuids':'uuid'},inplace=True)
+            
             for ibl_metric_key, property_name in ibl_metric_key_to_property_name.items():
-                all_unit_properties[property_name].extend(list(clusters["metrics"][ibl_metric_key]))
+                all_unit_properties[property_name].extend(list(cluster_metrics[ibl_metric_key]))
```

Comment:
Does it have to be a list here ? It's pretty sad not to be using a table or similar

Reply:
it was like this before, `all_unit_properties` is a dict of lists, unpacked again in line 140. Do you think it's a problem speed wise? Since the maximum number of entries is going to be 3k (2x neuropixel @ 1500 units) it will never be a large list ... in the BWM conversion at least.